packages/codemirror-graphql/src/hint.js

```diff
@@ -225,7 +225,7 @@ function getSuggestionsForFieldNames(
   if (typeInfo.parentType) {
     const parentType = typeInfo.parentType;
     const fields: GraphQLField<*, *>[] =
-      isObjectType(parentType) && 'getFields' in parentType
+      'getFields' in parentType
```

Comment:
It seems Flow doesn't understand `'getFields' in parentType` so errors is occurring in the next line by flow check. I'm thinking `isObjectType(parentType) || isInterfaceType(parentType)` without `getFields in parentType` is a suitable solution here because [GraphQLObjectType](https://github.com/graphql/graphql-js/blob/v15.0.0/src/type/definition.js#L701) and [GraphQLInterfaceType](https://github.com/graphql/graphql-js/blob/v15.0.0/src/type/definition.js#L1011) **must** have `getFields()` method so `getFields in parentType` is never reached. What do you think?

Reply:
That makes sense! Done!